### PR TITLE
feat: mark glitch item as edible

### PR DESCRIPTION
**Pull request by midgleyc**
Mark glitch item as edible, so "eat glitch season" works.

Unfortunately, this means that "use glitch season" does not work, even though the reward is marked as reusable. I don't know how to fix this, but would appreciate suggestions.

### CODE REVIEW

**Comment by libraryaddict** [comment]
First thing that pops to mind is adding `usable`, which to my mind at least; Should allow it to be `use`'d

**Comment by horrible-little-slime** [comment]
Would `eat` as a CLI command and an ash function need extra support to handle the incoming combat?

**Comment by midgleyc** [comment]
I don't think so, because it already works for black pudding (I'm pretty sure).

**Comment by midgleyc** [comment]
Gausie has fixed the issue. Thanks!

**Comment by gausie** [comment]
@midgleyc I think it's fair for you to review this since I've fiddled with it so much

File: test/net/sourceforge/kolmafia/textui/command/EatCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import net.sourceforge.kolmafia.KoLCharacter;
+import net.sourceforge.kolmafia.objectpool.ItemPool;
+import net.sourceforge.kolmafia.preferences.Preferences;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+class EatCommandTest extends AbstractCommandTestBase {
+  @BeforeEach
+  void beforeEach() {
```
Comment (midgleyc) [inline test/net/sourceforge/kolmafia/textui/command/EatCommandTest.java:21]: There's only one test so doesn't matter at the moment, but I think this should be able to be `BeforeAll` (with cleanups).

File: test/net/sourceforge/kolmafia/textui/command/UseCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+import net.sourceforge.kolmafia.KoLCharacter;
+import net.sourceforge.kolmafia.objectpool.ItemPool;
+import net.sourceforge.kolmafia.preferences.Preferences;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+class UseCommandTest extends AbstractCommandTestBase {
+  @BeforeEach
+  void beforeEach() {
```
Comment (midgleyc) [inline test/net/sourceforge/kolmafia/textui/command/UseCommandTest.java:21]: same `BeforeAll` comment here

**Review by midgleyc — COMMENTED** [review]
I can't approve or request changes on my own PR, but here are some changes.

Approve and merge when done ;)

**Review by gausie — APPROVED** [review]

**Pull request merged**